internal/pkg/log/level.go

```diff
@@ -0,0 +1,104 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+)
+
+type Level string
+
+const (
+	// Log Level [debug].
+	LevelDebug Level = "debug"
+	// Log Level [info].
+	LevelInfo Level = "info"
+	// Log Level [warn].
+	LevelWarn Level = "warn"
+	// Log Level [error].
+	LevelError Level = "error"
+	// Log Level [dpanic].
+	LevelDPanic Level = "dpanic"
+	// Log Level [panic].
+	LevelPanic Level = "panic"
+	// Log Level [fatal].
```

Comment:
These comments do not explain the levels. Please update so they communicate the logging strata they intend to enable.

Reply:
Added comments to levels